Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
```

Comment:
Consider using double quotes for the `go run main.go` command to maintain consistency across your commands. Currently, `run` uses single quotes while others use double quotes.